lib/core.ps1

```diff
@@ -750,26 +755,29 @@ function Confirm-InstallationStatus {
     $Apps | Select-Object -Unique | Where-Object { $_.Name -ne 'scoop' } | ForEach-Object {
         $App, $null, $null = parse_app $_
         if ($Global) {
-            if (installed $App $true) {
-                $Installed += ,@($App, $true)
-            } elseif (installed $App $false) {
-                error "'$App' isn't installed globally, but it is installed for your account."
+            if (Test-Path (appdir $App $true)) {
+                $Installed += , @($App, $true)
+            } elseif (Test-Path (appdir $App $false)) {
+                error "'$App' isn't installed globally, but it may be installed for your account."
```

Comment:
```suggestion
                error "'$App' isn't installed globally, but it may be installed locally."
```

I think `locally` is a better term than `account`.

Reply:
Changed